kubeflow/pipelines.py

```diff
@@ -79,6 +79,7 @@ def train_op(
     num_gpu,
     gpu_type,
     checkpoint_file=None,
+    train_split=None,
```

Comment:
[mustfix] We should allow user to pass the whole override string so that they can override arbitrary config from the kubeflow UI.

Reply:
Add override option in the pipeline. And tested in this [run](https://datasetinsights.endpoints.unity-ai-thea-test.cloud.goog/_/pipeline/#/runs/details/2ff13dcc-c1b2-4be8-b94d-0b28030a42c2).